SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -465,4 +479,6 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 	}
 
 	bool IsDeploymentConfigurationValid() const;
+
+	FORCEINLINE bool IsRunningInChina() const { return ServicesRegion == EServicesRegion::CN; }
```

Comment:
FYI @improbable-danny. @jessicafalk has merged a `SpatialGDKServicesSettings` file in her PR which will allow us to refactor this in the future. Could you spin up a ticket :D